src/init/highlights.ts

```diff
@@ -0,0 +1,54 @@
+import store from 'store';
+import { showViewer } from 'store/actions';
+import { mousePosition } from 'common/dom';
+import Highlighter from 'core/Highlighter';
+
+let instance;
```

Comment:
Why not just declare a ES6 class? Since it would be pretty much equivalent, I'd personally choose a class for readability. Do you find this pattern you're using more common/natural?

Reply:
This is a "flattened" (due to es6 modules(files)) version of the JS "module pattern", so it's a rather common one.